docs/contributing/Android-API-And-Related-Versions.md

```diff
@@ -0,0 +1,15 @@
+# Android Api and Related versions
+
+## Overview
+
+The most important thing is for any test, project, or app to have compatible versions. 
+If versions need to be different then a comment explaining why and what is being evaluated should be included. 
+All chosen versions should pass the minimum versions checks defined in [gradle_utils.dart](https://github.com/flutter/flutter/blob/a16c447abcb695b4ca907d59e66dc87f4f7178d3/packages/flutter_tools/lib/src/android/gradle_utils.dart#L63)
```

Comment:
s/should/must?

If this check is always made _for_ you (there is no action required), then let's make it clear we aren't asking anyone to do anything other than run the app (or the test).

Reply:
The only must is the top level, everything must work together which is enforced by tooling.